src/content/hcp/docs-landing.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
🗒️ Not sure why, but git didn't recognize this as just a file rename. The content _shouldn't_ be any different.